pkg/wdk/services_post_beef.go

```diff
@@ -0,0 +1,67 @@
+package wdk
+
+import (
+	"github.com/bsv-blockchain/go-sdk/transaction"
+	"github.com/go-softwarelab/common/pkg/seq"
+)
+
+// PostBeefResult is a list of results from the PostBEEF method of all services.
+type PostBeefResult []*PostBeefSingleResult
+
+// Success checks if one of the results is a success.
+func (it PostBeefResult) Success() bool {
+	return seq.Exists(seq.FromSlice(it), func(it *PostBeefSingleResult) bool {
+		return it.PostedBEEF != nil && it.Error == nil
+	})
+}
+
+// PostBeefSingleResult is the result of the PostBEEF method of a single service.
+// It contains the name of the service that produced the result and the result itself.
+// The result could be either a success or an error.
+type PostBeefSingleResult struct {
+	Name       string
+	PostedBEEF *PostedBEEF
```

Comment:
maybe `PostedBEEFResult` because PostedBEEF suggests that it contains the BEEF of transaction